lib/hapi-statsd.js

```diff
@@ -14,25 +14,25 @@ var StatsdClient = require('statsd-client'),
 		filters: [],
 	};
 
-module.exports.register = function(server, options, next) {
+var register = function (server, options) {
 	var settings = Hoek.applyToDefaults(defaults, options || {}),
 		statsdClient = options.statsdClient || new StatsdClient({
 			host: settings.host,
 			port: settings.port,
 			prefix: settings.prefix
 		}),
-		normalizePath = function(path) {
+		normalizePath = function (path) {
```

Comment:
No need to add these spaces but :shrug:. They aren't here in the original repo.

Reply:
Good callout. My editor added these spaces on its own but there aren't any eslint rules. 🤔 I checked some other functions and they do have a space in between. I am leaving this as is.